source/reporting/index.html.md.erb

```diff
@@ -0,0 +1,135 @@
+---
+title: Reporting 
+weight: 70
+---
+
+# Reporting 
+
+You can extract reporting information from GOV.UK Pay manually, or via the
+API.
+
+## Manual reporting
+
+You can sign in to your [GOV.UK Pay admin
+account](https://selfservice.payments.service.gov.uk/login) and export
+transaction information in a CSV file. 
+
+For a given account, select __Transactions__ in the __Dashboard__ page.
```

Comment:
I think "For a given account" isn't a great pattern for this particular context, in hindsight. It would be obvious to the user yes, but I think it would be better to say "For the account you want to look at". There may be a better way of saying it like, "For the account you want to extract reporting information for" but I'll let my PM comment on that when she's back if needed. Thanks for picking this up.